ansible/update_hosts.yml

```diff
@@ -103,6 +103,29 @@
       environment:
         PATH: /opt/privacyscore/.pyenv/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
         VIRTUAL_ENV: /opt/privacyscore/.pyenv
+    - name: Refresh privacyscanner git repository
+      become: yes
+      become_user: privacyscore
+      git:
+        repo: https://github.com/PrivacyScore/privacyscanner.git
+        dest: /opt/privacyscore/privacyscanner
+        version: master
+    - name: Install privacyscanner pip requirements
+      become: yes
+      become_user: privacyscore
+      pip:
+        editable: yes
+        name: /opt/privacyscore/privacyscanner
+      environment:
```

Comment:
The ansible `pip` module has a `virtualenv` parameter.

Reply:
I copy and pasted it from the other entries, but probably it's worth to rewrite them. I will do so in another commit for this branch...